app/components/hero_component.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class HeroComponent < ViewComponent::Base
+  def initialize(title, subtitle = '', schema = nil)
+    super
+    @title = title.length > 32 ? title.split.in_groups(2, false).map { |g| g.join(' ') }.join('<br> ').html_safe : title # rubocop:disable Rails/OutputSafety
```

Comment:
oh ruby. this prob needs moving into a method now i look at it again, lol

its basically to even out the line lengths on longer titles and avoid widows